xml/vm_security.xml

```diff
@@ -0,0 +1,202 @@
+      <dm:translation>yes</dm:translation>
+    </dm:docmanager>
+  </info>
+  <sect1 xml:id="vm-security-hardware-support">
+    <title>Supported Hardware</title>
+
+    <para>
+      A system with an AMD EPYC (3rd Gen or newer) is required run AMD SEV-SNP virtual machines. The BIOS of the AMD machine must provide the necessary options to enable support for confidential computing on the platform.</para>
+  </sect1>
+  <sect1 xml:id="vm-security-enable-confidential-compute-module">
+    <title>Enabling Confidential Compute Module</title>
+
+    <para>
+      Support for AMD SEV-SNP is available as a Technology Preview in SUSE Linux Enterprise Server 15-SP6. However, the necessary packages are not part of the default installation or repositories.</para>
+    <para>The packages are shipped via Confidential Compute Module. You must enable it at system installation time or later via the SUSEConnect command line tool.</para>
+    <itemizedlist>
+      <listitem>
+        <para>To check whether the module is already enabled, run the command:
+        </para>
+        <screen># suseconnect -l</screen>
+        <para>This displays the list of available modules with their activation status and commands to enable the inactive modules.</para>
+      
+      <para>The inactive confidential compute module appears as given below:</para>
+    <screen>Confidential Computing Technical Preview Module 15 SP6 x86_64
+      Activate with: suseconnect -p sle-module-confidential-computing/15.6/x86_64</screen>
+    </listitem>
+  <listitem> <para>To enable the Confidential Computing Technical Preview Module, run the command:</para>
+    <screen># suseconnect -p sle-module-confidential-computing/15.6/x86_64
+      Registering system to SUSE Customer Center
+      
+      Updating system details on https://scc.suse.com ...
+      
+      Activating sle-module-confidential-computing 15.6 x86_64 ...
+      Adding service to system ...
+      Installing release package ...
+      
+      Successfully registered system</screen>
+      <para>The confidential compute module is enabled and you can install the packages.</para>
+    </listitem>
+    </itemizedlist>
+  </sect1>
+  <sect1 xml:id="vm-security-verify-setup">
+    <title>Installing Packages and Setting up the Base System</title>
+
+    <para>
+      The confidential compute module provides replacement packages supporting AMD SEV-SNP. To ensure a maximum of compatibility, these packages are based on the code streams from SUSE Linux Enterprise Server.</para>
+<para>The three components that need to be replaced are:</para>
+<itemizedlist>
+  <listitem>
+    <para>The Linux kernel</para>
+  </listitem>
+<listitem>
+  <para>QEMU Virtual Machine Monitor</para>
+</listitem>
+<listitem>
+  <para>&libvirt; framework</para>
+</listitem>
+</itemizedlist>
+<procedure>
+  <step>
+    <para>To install the replacement packages, run the command:</para>
+    <screen># sudo zypper install coco:kernel-coco coco:qemu coco:libvirt
```

Comment:
The repo name is not coco; This needs to be adjusted with the name of the CoCo repository.